source/leuven/tier2_hardware/wice_hardware.rst

```diff
@@ -26,17 +42,44 @@ Hardware details
     (2 NUMA domains and 1 L3 cache per CPU)
   - 2048 GiB RAM
   - 960 GB SSD local disk
-  - partition ``bigmem``, :ref:`submit options <submit to wice big memory node>`
+  - partition ``bigmem``,
+    :ref:`submit options <submit to wice big memory node>`
 
-- 4 GPU nodes, 16 GPU devices
+- 1 huge memory node
 
-  - 2 Intel Xeon Platinum 8360Y CPUs\@2.4 GHz (Ice lake),
+  - 2 Intel Xeon Platinum 8360Y CPUs (Ice lake),
     36 cores each :raw-html:`<br />`
-    (2 NUMA domains and 1 L3 cache per CPU)
-  - 512 GiB RAM
-  - 4 NVIDIA A100 SXM4, 80 GiB GDDR, connected with NVLink
+    (1 NUMA domain and 1 L3 cache per CPU) :raw-html:`<br />`
+    The base and max CPU frequencies are 2.4 GHz and 3.5 GHz, respectively.
+  - 8 TiB RAM
   - 960 GB SSD local disk
-  - partition ``gpu``, :ref:`submit options <submit to wice GPU node>`
+  - partition ``hugemem``,
+    :ref:`submit options <submit to wice big memory node>`
+
+- 8 GPU nodes
+
+  - 4 nodes with 16 A100 GPUs in total
+
+    - 2 Intel Xeon Platinum 8360Y CPUs\@2.4 GHz (Ice lake),
+      36 cores each :raw-html:`<br />`
+      (2 NUMA domains and 1 L3 cache per CPU)
+    - 512 GiB RAM
+    - 4 NVIDIA A100 SXM4, 80 GiB GDDR, connected with NVLink
+    - 960 GB SSD local disk
+    - partition ``gpu``,
+      :ref:`submit options <submit to wice GPU node>`
+
+  - 4 nodes with 16 H100 GPUs in total
+
+    - 2 AMD EPYC 9334 CPUs (Genoa),
+      32 cores each :raw-html:`<br />`
+      (4 NUMA domains and 4 L3 caches per CPU) :raw-html:`<br />`
+      The base and max CPU frequencies are 2.7 GHz and 3.9 GHz, respectively.
+    - 768 GiB RAM
+    - 4 NVIDIA H100 SXM4, 80 GiB HBM3, connected with NVLink
```

Comment:
It's H100 SXM5 instead of 4.

Reply:
👍 (done in 5144743c39a6a0cffcae850847639a7e58ab5170)